test/table.js

```diff
@@ -1438,6 +1438,19 @@ describe('Bigtable/Table', function() {
           done();
         });
       });
+
+      it('should make the correct request', function(done) {
```

Comment:
`getRows` doesn't actually make this request, so this test is misplaced. We should only have tests under the `describe` block that matches the method's name.